exercises/hello-world/makefile

```diff
@@ -4,10 +4,17 @@ CFLAGS += -Wextra
 CFLAGS += -pedantic
 CFLAGS += -Werror
 
+VFLAGS  = --quiet
+VFLAGS += --tool=memcheck
+VFLAGS += --leak-check=full
+VFLAGS += --error-exitcode=1
 
 test: tests.out
 	@./tests.out
 
+memcheck: tests.out
+	@valgrind $(VFLAGS) ./tests.out >> /dev/null
```

Comment:
Should we echo something hhere so users knows something happened.  Otherwise with --quiet it just returns.